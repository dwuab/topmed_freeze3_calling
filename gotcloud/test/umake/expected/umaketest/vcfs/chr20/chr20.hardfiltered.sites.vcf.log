Analysis started on <date>

The following filters are in effect:
m20 : MQ >= 20.00
dp60 : DP >= 60.00
DP60000 : DP <= 60000.00
ns30 : NS >= 30.00
AB70 : AB <= 0.70
STR20 : STR <= 0.20
str-20 : STR >= -0.20
stz-5 : STZ >= -5.00
STZ5 : STZ <= 5.00
CBR20 : CBR <= 0.20
AOI5 : AOI <= 5.00
MQ0_10 : MQ0 <= 0.10
fic-20 : FIC >= -0.20
LQR30 : LQR <= 0.30
q5 : QUAL >= 5
INDEL5 : INDEL >= 5 bp with <gotcloud_root>/test/chr20Ref/1kg.pilot_release.merged.indels.sites.hg19.chr20.vcf

Writing to VCF file <outdir_path>/vcfs/chr20/chr20.hardfiltered.sites.vcf
Analysis finished on <date>

