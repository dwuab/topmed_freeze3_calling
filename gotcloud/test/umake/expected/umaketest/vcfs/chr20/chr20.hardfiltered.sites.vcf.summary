--------------------------------------------------------------------------------------------------------------
              FILTER	#SNPs	#dbSNP	%dbSNP	%CpG	%CpG	%Known	%Novel	%nCpG-K	%nCpG-N	%HM3	%HM3
                    				Known	Novel	Ts/Tv	Ts/Tv	Ts/Tv	Ts/Tv	sens	/SNP
--------------------------------------------------------------------------------------------------------------
                AB70	7	4	57.1	0.0	0.0	3.00	2.00	3.00	2.00	0.003	14.286
  AB70;MQ0_10;fic-20	1	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
          AB70;STR20	2	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
     AB70;STR20;STZ5	2	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
AB70;STR20;STZ5;CBR20	3	0	0.0	NA	0.0	NA	2.00	NA	2.00	0.000	0.000
AB70;STR20;STZ5;CBR20;fic-20	2	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
AB70;STR20;STZ5;fic-20	1	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
         AB70;fic-20	1	1	100.0	0.0	NA	NA	NA	NA	NA	0.000	0.000
         AB70;str-20	1	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
  AB70;str-20;fic-20	1	0	0.0	NA	0.0	NA	NA	NA	NA	0.000	0.000
   AB70;str-20;stz-5	1	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
AB70;str-20;stz-5;CBR20	1	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
AB70;str-20;stz-5;CBR20;fic-20	2	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
                AOI5	1	1	100.0	100.0	NA	NA	NA	NA	NA	0.000	0.000
               CBR20	15	15	100.0	13.3	NA	2.75	NA	3.33	NA	0.011	26.667
        CBR20;MQ0_10	5	5	100.0	0.0	NA	4.00	NA	4.00	NA	0.000	0.000
              INDEL5	9	8	88.9	0.0	100.0	NA	0.00	NA	NA	0.003	11.111
        INDEL5;CBR20	1	1	100.0	0.0	NA	NA	NA	NA	NA	0.000	0.000
       INDEL5;fic-20	1	1	100.0	0.0	NA	NA	NA	NA	NA	0.000	0.000
              MQ0_10	65	58	89.2	29.3	0.0	3.46	NA	2.42	NA	0.000	0.000
       MQ0_10;fic-20	7	6	85.7	16.7	0.0	2.00	NA	1.50	NA	0.000	0.000
                PASS	932	905	97.1	15.4	3.7	2.42	2.00	2.21	1.89	0.544	21.674
               STR20	5	3	60.0	33.3	0.0	0.50	0.00	1.00	0.00	0.003	20.000
STR20;CBR20;MQ0_10;fic-20	1	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
        STR20;MQ0_10	1	0	0.0	NA	0.0	NA	NA	NA	NA	0.000	0.000
          STR20;STZ5	1	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
    STR20;STZ5;CBR20	3	0	0.0	NA	33.3	NA	0.00	NA	0.00	0.000	0.000
STR20;STZ5;CBR20;MQ0_10	3	0	0.0	NA	0.0	NA	0.50	NA	0.50	0.000	0.000
STR20;STZ5;CBR20;MQ0_10;fic-20	2	1	50.0	0.0	0.0	0.00	NA	0.00	NA	0.000	0.000
STR20;STZ5;CBR20;fic-20	1	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
   STR20;STZ5;fic-20	1	0	0.0	NA	0.0	NA	NA	NA	NA	0.000	0.000
        STR20;fic-20	2	1	50.0	0.0	0.0	NA	0.00	NA	0.00	0.000	0.000
   dp60;CBR20;MQ0_10	1	1	100.0	0.0	NA	NA	NA	NA	NA	0.000	0.000
         dp60;MQ0_10	1	1	100.0	0.0	NA	NA	NA	NA	NA	0.000	0.000
dp60;ns30;CBR20;AOI5	1	1	100.0	0.0	NA	0.00	NA	0.00	NA	0.000	0.000
dp60;ns30;CBR20;MQ0_10	1	1	100.0	0.0	NA	NA	NA	NA	NA	0.000	0.000
    dp60;ns30;MQ0_10	3	3	100.0	0.0	NA	0.50	NA	0.50	NA	0.000	0.000
dp60;ns30;MQ0_10;fic-20	1	0	0.0	NA	0.0	NA	NA	NA	NA	0.000	0.000
              fic-20	10	9	90.0	0.0	0.0	1.25	NA	1.25	NA	0.011	40.000
              str-20	3	0	0.0	NA	0.0	NA	0.50	NA	0.50	0.000	0.000
        str-20;LQR30	1	0	0.0	NA	0.0	NA	NA	NA	NA	0.000	0.000
       str-20;MQ0_10	3	2	66.7	50.0	0.0	NA	NA	NA	NA	0.000	0.000
       str-20;fic-20	3	1	33.3	0.0	0.0	NA	1.00	NA	1.00	0.000	0.000
        str-20;stz-5	3	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
  str-20;stz-5;CBR20	2	0	0.0	NA	0.0	NA	0.00	NA	0.00	0.000	0.000
 str-20;stz-5;fic-20	3	2	66.7	0.0	0.0	NA	0.00	NA	0.00	0.000	0.000
--------------------------------------------------------------------------------------------------------------
              FILTER	#SNPs	#dbSNP	%dbSNP	%CpG	%CpG	%Known	%Novel	%nCpG-K	%nCpG-N	%HM3	%HM3
                    				Known	Novel	Ts/Tv	Ts/Tv	Ts/Tv	Ts/Tv	sens	/SNP
--------------------------------------------------------------------------------------------------------------
                AB70	25	5	20.0	0.0	0.0	4.00	0.33	4.00	0.33	0.003	4.000
                AOI5	2	2	100.0	50.0	NA	1.00	NA	0.00	NA	0.000	0.000
               CBR20	44	25	56.8	8.0	5.3	2.57	0.27	2.83	0.29	0.011	9.091
              INDEL5	11	10	90.9	0.0	100.0	NA	0.00	NA	NA	0.003	9.091
               LQR30	1	0	0.0	NA	0.0	NA	NA	NA	NA	0.000	0.000
              MQ0_10	95	78	82.1	24.4	0.0	3.11	3.25	2.28	3.25	0.000	0.000
                PASS	932	905	97.1	15.4	3.7	2.42	2.00	2.21	1.89	0.544	21.674
               STR20	30	5	16.7	20.0	4.0	0.67	0.32	1.00	0.33	0.003	3.333
                STZ5	19	1	5.3	0.0	5.6	0.00	0.38	0.00	0.42	0.000	0.000
                dp60	8	7	87.5	0.0	0.0	1.33	NA	1.33	NA	0.000	0.000
              fic-20	40	22	55.0	4.5	0.0	2.14	0.64	2.00	0.64	0.011	10.000
                ns30	6	5	83.3	0.0	0.0	0.67	NA	0.67	NA	0.000	0.000
              str-20	24	5	20.8	20.0	0.0	NA	0.36	NA	0.36	0.000	0.000
               stz-5	12	2	16.7	0.0	0.0	NA	0.00	NA	0.00	0.000	0.000
--------------------------------------------------------------------------------------------------------------
                PASS	932	905	97.1	15.4	3.7	2.42	2.00	2.21	1.89	0.544	21.674
                FAIL	185	126	68.1	18.3	3.4	3.06	0.64	2.68	0.68	0.030	5.946
               TOTAL	1117	1031	92.3	15.7	3.5	2.48	0.91	2.25	0.93	0.573	19.069
------------------------------------------------------------------------------------------------------------
 MultiAllele Ref/Alt	0
  Repeated Positions	0
       TOTAL SKIPPED	0
------------------------------------------------------------------------------------------------------------
