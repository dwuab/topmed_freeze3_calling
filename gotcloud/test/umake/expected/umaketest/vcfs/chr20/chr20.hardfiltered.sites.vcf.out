
Available Options
                         Recipes : --write-bed, --write-vcf [ON], --upgrade,
                                   --summarize, --filter [ON], --subset
               VCF Input options : --in-vcf [<outdir_path>/vcfs/chr20/chr20.merged.stats.vcf]
               BED Input options : --in-bfile [], --in-bed [], --in-bim [],
                                   --in-fam [],
                                   --ref [/data/local/ref/karma.ref/human.g1k.v37.fa]
              Subsetting options : --in-subset [], --mono-subset,
                                   --filt-only-subset
                  Output Options : --out [<outdir_path>/vcfs/chr20/chr20.hardfiltered.sites.vcf],
                                   --qGeno, --print-every [10000]
      Output compression Options : --plain [ON], --bgzf, --gzip
   Genotype-level Filter Options : --minGQ, --minGD
                  Filter Options : --winIndel [5],
                                   --indelVCF [<gotcloud_root>/test/chr20Ref/1kg.pilot_release.merged.indels.sites.hg19.chr20.vcf],
                                   --minQUAL [5], --minMQ [20],
                                   --maxDP [60000], --minDP [60],
                                   --maxABL [70], --winFFRQ, --maxFFRQ,
                                   --winFVAR, --merFVAR, --maxFVAR,
                                   --minNS [30], --maxSTP [2147483647],
                                   --maxTTT [2147483647],
                                   --minTTT [-2147483648], --maxSTR [20],
                                   --minSTR [-20], --maxSTZ [5], --minSTZ [-5],
                                   --maxCBR [20], --minCBR [-100],
                                   --maxQBR [100], --minQBR [-100],
                                   --maxCBZ [2147483647], --maxCSR [100],
                                   --minCSR [-100], --maxLQZ [2147483647],
                                   --minLQZ [-2147483648],
                                   --maxRBZ [2147483647],
                                   --minRBZ [-2147483648],
                                   --maxIOZ [2147483647],
                                   --minIOR [-2147483648],
                                   --maxIOR [2147483647],
                                   --maxAOZ [2147483647], --maxAOI [5],
                                   --maxMQ0 [10], --maxMQ10 [100],
                                   --maxMQ20 [100], --minFIC [-20],
                                   --minABE [-100], --maxABE [100],
                                   --maxLQR [30], --minMBR [-100],
                                   --maxMBR [100], --minABZ [-2147483648],
                                   --maxABZ [2147483647],
                                   --maxBCS [2147483647], --keepFilter

vcfCooker 1.1.1 -- Manipulate VCF files
(c) 2010-2014 Hyun Min Kang, Matthew Flickinger, Matthew Snyder, Paul Anderson, Tom Blackwell, Mary Kate Wing, and Goncalo Abecasis

